ics-frontend/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import {GalleryComponent} from './gallery/gallery.component';
+import { SingleImagePageComponent } from './single-image-page/single-image-page.component';
+import { ImageClassificationComponent } from './image-classification/image-classification.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {path: 'gallery', component: GalleryComponent},
+  {path: 'image-classification', component: ImageClassificationComponent},
```

Comment:
I think you can delete this one

Reply:
@asalov I thought that because it calls the ImageClassificationComponent, which is different than the SingleImagePageComponent I would need both. Could you explain why I would not need it?

Reply:
I saw your other comment and understood what your idea was so you can ignore this